management/user.go

```diff
@@ -340,3 +368,20 @@ func (m *UserManager) Blocks(id string) ([]*UserBlock, error) {
 func (m *UserManager) Unblock(id string) error {
 	return m.delete(m.uri("user-blocks", id))
 }
+
+// GenerateRecoveryCode removes the current multi-factor authentication recovery code and generate a new one.
+//
+// See: https://auth0.com/docs/api/management/v2#!/Users/post_recovery_code_regeneration
+func (m *UserManager) GenerateRecoveryCode(id string) (*UserRecoveryCode, error) {
+	r := new(UserRecoveryCode)
+	err := m.post(m.uri("users", id, "recovery-code-regeneration"), r)
+	return r, err
+}
+
+// ListEnrollments retrieves all Guardian enrollments for a user.
+//
+// See: https://auth0.com/docs/api/management/v2#!/Users/get_enrollments
+func (m *UserManager) ListEnrollments(id string) (enrols []*UserEnrollment, err error) {
+	err = m.get(m.uri("users", id, "enrollments"), &enrols)
+	return enrols, err
+}
```

Comment:
```suggestion
// Enrollments retrieves all Guardian enrollments for a user.
//
// See: https://auth0.com/docs/api/management/v2#!/Users/get_enrollments
func (m *UserManager) Enrollments(id string) (enrolls []*UserEnrollment, err error) {
	err = m.get(m.uri("users", id, "enrollments"), &enrolls)
	return enrolls, err
}
```

This should make it similar to other sub resource list methods such as `Blocks(...)`, `Permissions(...)`, `Roles(...)` etc.